rasa/train.py

```diff
@@ -157,6 +188,18 @@ async def _train_async_internal(
         file_importer.get_stories(), file_importer.get_nlu_data()
     )
 
+    # get the fingerprint
+    new_fingerprint = await model.model_fingerprint(file_importer)
+    old_model = model.get_latest_model(output_path)
+
+    if dry_run:
+        fingerprint_comparison = model.should_retrain(
```

Comment:
In this case we can drop calling this again in `230` (if `force_training` is `True`, I'd probably return a `3` as exit code?)

Reply:
Done!